parmed/amber/titratable_residues.py

```diff
@@ -482,7 +519,7 @@ def write_cpin(self, output, igb=2, intdiel=1.0, oldfmt=False, typ="ph", coions=
 
 AS4 = TitratableResidue('AS4', ['N', 'H', 'CA', 'HA', 'CB', 'HB2', 'HB3', 'CG',
                         'OD1', 'OD2', 'HD21', 'C', 'O', 'HD22', 'HD11', 'HD12'],
-                        pka=4.0, typ="ph")
```

Comment:
Where does the pka adjustment happen if you got rid of this parameter but didn’t change the reference energy?

I’m nervous that this could break cpinutil in subtle ways.

Reply:
Good question @swails . This variable pka was only used in two places: during printing when someone types "cpinutil.py --describe" or as an identifier when someone uses the minpka or maxpka options in cpinutil.py (these options were removed). As you can see in the code, the set_pKa calls are still in place, so the pka adjustment when the old format is requested is sill going to work fine. You may check my amber branch vwdc_phe-merge and you will see that the cpinutil.py tests are passing (and there are tests with the old format).

I had to remove this pka variable because when we move to the more general cases, more than one pka values might be associated to the same residue, or, with these new adaptations, multiple pka and Eo values could be associated with the same residue.

Please let me know what you think.

Reply:
That was already the case with Histidine. I just took the macroscopic pKa for the side-chain in general. I think those options were helpful for doing, for instance, a base-range or acid-range titration. Users didn’t need to know *which* residues were titratable at which standard pH ranges. They could just say what residues they wanted to be titratable by giving a general range of solvent conditions. 

I think those are options worth preserving.

Reply:
Ok. I will bring these features back.